bentoml/adapters/base_input.py

```diff
@@ -89,7 +106,7 @@ def from_inference_job(self, *_args, **_kwargs) -> Iterator[InferenceTask]:
         raise NotImplementedError()
 
     def extract_user_func_args(
-        self, tasks: Iterable[InferenceTask]
+        self, tasks: Iterable[InferenceTask], validation: bool = True,
```

Comment:
is it possible to do it without changing the signature of `extract_user_func_args`?

Many users are using custom input types by creating a subclass from the BaseInput, and doing this will be a breaking change for them.

How about we make the tope level `validation.enabled` config under bento_service, and inject it at base_input's `__init__` method?

```
bento_service:
   input_validation:
        enabled: True
```